Dockerfile.potree

```diff
@@ -1,9 +1,9 @@
-FROM python:3.7-slim
+FROM pdal/pdal:2.1
```

Comment:
This does increase the size of the image.  We could invest time into building PDAL in our image.  I could open a jira issue for that if needed.